blog/2021-03-16-BatchSolvingWithAmq.adoc

```diff
@@ -0,0 +1,182 @@
+= Batch solving in a stateless manner with OptaPlanner and AMQ
+:page-interpolate: true
+:imagesdir: 2021/03/16
+:awestruct-author: rsynek
+:awestruct-layout: blogPostBase
+:awestruct-tags: [amq, quarkus, messaging, integration]
+:awestruct-share_image_filename: architecture.png
+
+Let's take a look at a new https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/amq-quarkus-school-timetabling[quickstart], which shows how to build a scalable, stateless service capable of solving numerous instances of the same planning problem.
+This post will be most interesting to developers looking for how to bring OptaPlanner from an embedded engine to a service communicating with other applications.
```

Comment:
"will be most interesting" -> "is interesting" (active, non-verbose voice)
"This post is interesting to developers looking for how to ..." -> "This article shows how to ..."
```suggestion
This article show how to expose an multiple OptaPlanner nodes as an AMQ queue to which other applications can submit datasets to solve.
```
St

Reply:
This sentence is now maybe obsolete